app/models/tag.rb

```diff
@@ -443,7 +445,7 @@ def nids
   end
 
   def span(start, fin)
-    start.to_date.step(fin.to_date, 7).count
+    start.to_date.step(fin.to_date, 7).size
```

Comment:
hello, seems a couple of things won't work here. Lets leave this as `.count`  `.size` will return nil here